odl/phantom/transmission.py

```diff
@@ -317,9 +317,9 @@ def transposeravel(arr):
     # Rescale points to the default grid.
     # The forbild phantom is defined on [-12.8, 12.8] x [-12.8, 12.8]
     xcoord, ycoord = space.points().T
-    xcoord = (xcoord - np.min(xcoord)) / (np.max(xcoord) - np.min(xcoord))
+    xcoord = (xcoord - space.min_pt[0]) / (space.max_pt[0] - space.min_pt[0])
     xcoord = 25.8 * xcoord - 12.8
-    ycoord = (ycoord - np.min(ycoord)) / (np.max(ycoord) - np.min(ycoord))
+    ycoord = (ycoord - space.min_pt[1]) / (space.max_pt[1] - space.min_pt[1])
```

Comment:
We really need `.grid` here in between, now you get the partition min/max instead of the grid min/max.
```python
>>> space = odl.uniform_discr(0, 1, 5)
>>> space.min_pt
array([ 0.])
>>> space.grid.min_pt
array([ 0.1])
>>> np.min(space.points())
0.10000000000000001
```